sumpy/tools.py

```diff
@@ -769,6 +771,9 @@ def _allow_redundant_execution_of_knl_scaling(knl):
                 knl, within=ObjTagged(ScalingAssignmentTag()))
 
 
+KernelCacheWrapper = KernelCacheMixin
```

Comment:
Have a deprecation based on module `__getattr__`? Feel free to bump the compat target to 3.8 to avoid having to do a 3.6 fallback.

Reply:
Added in 
https://github.com/inducer/sumpy/compare/e5ed7b5666c6b6b5ab3cfe838ac1f004bec4ef35..a091c5991fc9c2c3e85eff978bba99f700a3d67e